awsutils/cloudformation.py

```diff
@@ -61,3 +67,8 @@ def get_stack_instances(self, stack_name_or_id):
     def get_stack_instance_ids(self, stack_name_or_id):
         return [
             x.instance_id for x in self.get_stack_instances(stack_name_or_id)]
+
+    def stack_deleted(self, stack_name_or_id):
+        ''' Returns True if stack not found'''
+        stacks = self.conn_cfn.describe_stacks()
+        return stack_name_or_id not in [s.stack_name for s in stacks]
```

Comment:
And minor point but it would be much clearer if we named this method `stack_exists` and invert the return - this meth doesn't actually have anything to do with deleting (other than how we are using it right now)